docs/release-notes/release-2024-05-07.md

```diff
@@ -0,0 +1,157 @@
+---
+title: 0.56 - May 2024
+---
+
+# Release Notes Version 0.56 - May 2024
+
+:::warning
+This release contains deprecations and breaking changes
```

Comment:
I am missing the deprecations

Reply:
ah no that was from before when I had the value changes described too